pypesto/select/__init__.py

```diff
@@ -0,0 +1,19 @@
+"""
+Model Selection
+===============
+
+Perform model selection with a PEtab Select problem.
+"""
+
+from .misc import model_to_pypesto_problem
+from .problem import Problem
+
+try:
```

Comment:
A problem with this style (I guess we use it also elsewhere) is that now you have pypesto.select.petab_select as a submodule, which you probably don't intend. To avoid making petab_select public here, you can just move this check to some file where it's called.

Reply:
Added the following, should be OK now?
```python
del petab_select
```

Reply:
haha cheat but also fine